web/ViewModels/PropertyViewModel.cs

```diff
@@ -6,22 +6,22 @@ namespace BricksAndHearts.ViewModels;
 
 public class PropertyViewModel
 {
-    [Required] public PropertyAddress Address { get; set; }
+    [Required] public PropertyAddress? Address { get; set; }
```

Comment:
Can we try to address some of these warnings with default values as opposed to making them nullable? Especially this Address field so that we can get rid of the non-null assertions in the view.

Reply:
You can set this to a default Address object with `= new();` 🙂

Reply:
Done!